### PR TITLE
chore: update 2021-12-23 news

### PR DESCRIPTION
**Pull request by bfabio**

### CODE REVIEW

File: _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md [hunk 1]
```diff
@@ -10,18 +10,19 @@ image: /assets/images/posts/2021-12-23/buone-feste-it.png
...
+Un'attività che continuerà e diventerà sempre più intensa a partire dal 2022.
 
-Un'attività che continuerà e diventerà sempre più forte e visibile a partire dal 2022. In questi giorni, infatti, è in avvio una **completa revisione del *design system* del Paese**, ovvero di quegli strumenti necessari alla realizzazione di siti e servizi digitali della Pubblica Amministrazione **coerenti, familiari e fruibili per chiunque**. Nella pratica, schemi, regole e librerie visive e tecnologiche condivise. 
+In questi giorni, infatti, **è in avvio una completa revisione del *design system* del Paese**, ovvero degli strumenti necessari alla realizzazione di siti e servizi digitali della Pubblica Amministrazione, così da renderli coerenti, familiari e fruibili da tutti i cittadini. Una revisione che, nella pratica, si tradurrà in schemi, regole e librerie visive e tecnologiche condivise.
```
Comment (libremente) [inline _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md:17]: `regole e librerie visive e tecnologiche condivise` frase non chiara IMHO

File: _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md [hunk 2]
```diff
@@ -10,18 +10,19 @@ image: /assets/images/posts/2021-12-23/buone-feste-it.png
...
 
-Un’operazione ambiziosa, che prevede l’ammodernamento in termini **tecnologici** e di **accessibilità**, ma anche di **visione** e **gestione**, delle risorse messe a disposizione da Designers Italia e Developers Italia, in particolare i due kit di [progettazione](http://designers.italia.it/kit/progettazione-interfaccia/) e [sviluppo](https://designers.italia.it/kit/sviluppo-interfaccia/) interfaccia, dei principali esempi di applicazione e dei [modelli standard di siti web per le PA (Comuni e scuole)](https://designers.italia.it/modelli/).
+Si tratta di un’operazione ambiziosa, che prevede **l’ammodernamento in termini tecnologici e di accessibilità**, ma anche di visione e gestione, delle risorse messe a disposizione da Designers Italia e Developers Italia, in particolare i due kit di [progettazione](http://designers.italia.it/kit/progettazione-interfaccia/) e [sviluppo interfaccia](https://designers.italia.it/kit/sviluppo-interfaccia/), dei principali esempi di applicazione e dei [modelli standard di siti web per le PA](https://designers.italia.it/modelli/) (Comuni e scuole).
```
Comment (libremente) [inline _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md:19]: Vedo un po' di apostrofi sbagliati ma magari è tutto WIP

Comment (bfabio) [reply _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md:19]: Dove?

File: _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md [hunk 3]
```diff
@@ -10,18 +10,19 @@ image: /assets/images/posts/2021-12-23/buone-feste-it.png
...
+Jingle bells! È tempo di auguri e di bilanci per Developers Italia e [Designers Italia](https://designers.italia.it/). L’anno appena trascorso ci ha visti impegnati a promuovere, spesso lontano dai riflettori, un **cambiamento culturale** presso tutti gli stakeholder che contribuiscono alla trasformazione digitale della Pubblica Amministrazione.
+Un percorso che ha coinvolto migliaia di esperte ed esperti in progettazione e sviluppo, insieme a funzionari della PA, grazie a community call, corsi di formazione e collaborazioni dirette sui nostri canali di lavoro ([Slack](https://slack.developers.italia.it), [Forum Italia](https://forum.italia.it) e [GitHub](https://github.com/italia)).
+Un'attività che continuerà e diventerà sempre più intensa a partire dal 2022.
```
Comment (libremente) [inline _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md:15]:
```suggestion
Un’attività che continuerà e diventerà sempre più intensa a partire dal 2022.
``

**Review by libremente — CHANGES_REQUESTED** [review]

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Update _posts/it/2021-12-23-2022-cominciano-le-prime-sfide.md

Co-authored-by: libremente <surf@libremente.eu>
- subject may not be empty
- type may not be empty
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 933ed13d8d288ffe6e78e3dafbb3387848a3c8bf
</p>

**Review by libremente — APPROVED** [review]

**Pull request merged**